File: skills/astra-toolkit/clients/java/examples/tables-drop-index-exists.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;
import com.datastax.astra.client.tables.commands.options.DropTableIndexOptions;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN")).getDatabase(System.getenv("ASTRA_DB_API_ENDPOINT"));

    // Drop an index
    DropTableIndexOptions options = new DropTableIndexOptions().ifExists(true);

    database.dropTableIndex("**INDEX_NAME**", options);
  }
}

File: skills/astra-toolkit/clients/csharp/examples/tables-drop-index-exists.cs
using DataStax.AstraDB.DataApi;
using DataStax.AstraDB.DataApi.Tables;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Get an existing database
    var client = new DataAPIClient();
    var database = client.GetDatabase(
      System.Environment.GetEnvironmentVariable("ASTRA_DB_API_ENDPOINT"),
      System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN")
    );

    // Drop an index
    await database.DropTableIndexAsync(
      "**INDEX_NAME**",
      new DropTableIndexOptions() { IfExists = true }
    );
  }
}
